cumulus_library/studies/core/manifest.toml

```diff
@@ -36,4 +37,5 @@ export_list = [
     "core__count_condition_month",
     "core__count_medicationrequest_month",
     "core__meta_date",
+    "core__meta_version",
```

Comment:
This might be a dumb question - but these meta tables are for the whole study, right? Why not just have one meta table with columns like `date` and `version`? I could imagine either approach making your life easier, so I assume that sort of reason is why.

Reply:
I guess my thought was this: these are both quasi-optional, and may be present or absent, and this way there isn't a table schema dependency issue if we ever have to touch one or the other individually?